src/styles/utils/_vars.scss

```diff
@@ -1 +1,16 @@
-$c-gray: #eee;
+$transitionTime: 0.3s;
+
+$c-1: #0db2b3;
+$c-2: #131313;
+$c-3: #333;
+$c-4: #4f4f4f;
+$c-5: #7c7c7c;
+$c-6: #bdbdbd;
+$c-7: #f7f7f7;
+$c-8: #39bebf;
+$c-9: #0c797a;
+$c-10: #d6ecec;
+$c-11: #bdbdbd;
+
+$screen-tablet: 744px;
+$screen-desktop: 1280px;
```

Comment:
Use the same style of writing variables.
A little above, you wrote in this `transitionTime` format, and these are already hyphenated.